<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task8</class>
 <widget class="QMainWindow" name="Task8">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task6</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>531</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>590</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="timeBox">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>590</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget_Imp" native="true">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>471</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>600</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>600</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>steps on x</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1155</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
